.github/workflows/gcp-deploy.yaml

```diff
@@ -48,158 +70,192 @@ jobs:
+          echo "::set-output name=BACKEND_HOST::\
+          ${{env.BACKEND_SERVICE}}-${{env.CLOUDRUN_SUFFIX}}.${{env.DEPLOYMENT_DOMAIN}}"
+          echo "::set-output name=FRONTEND_URL::\
+          https://${{env.FRONTEND_SERVICE}}-${{env.CLOUDRUN_SUFFIX}}.${{env.DEPLOYMENT_DOMAIN}}"
+          echo "::set-output name=DB_PASSWORD::$(head -c 48 /dev/urandom | tr -cd [:alnum:])"
 
       - name: Those Variables May Now Be Interpolated
         id: second
         run: |
-          echo "::set-output name=HASURA_HOST::${{steps.first.outputs.HASURA_SERVICE}}-${{env.CLOUDRUN_SUFFIX}}.${{env.DEPLOYMENT_DOMAIN}}"
-          echo "::set-output name=DOCKER_PATH::${{steps.first.outputs.DOCKER_REGISTRY}}/${{env.PROJECT_ID}}/${{env.REGISTRY_REPO}}"
-          echo "::set-output name=SC_MIGRATE_URL::https://${{steps.first.outputs.BACKEND_HOST}}/actions/migrateSourceCredAccounts?force=true"
-
-      - name: And Again In Another Step
+          echo "::set-output name=HASURA_HOST::\
+          ${{steps.first.outputs.HASURA_SERVICE}}-${{env.CLOUDRUN_SUFFIX}}.${{env.DEPLOYMENT_DOMAIN}}"
+          echo "::set-output name=DOCKER_PATH::\
+          ${{steps.first.outputs.DOCKER_REGISTRY}}/${{env.PROJECT_ID}}/${{env.REGISTRY_REPO}}"
+          echo "::set-output name=SC_MIGRATE_URL::\
+          https://${{steps.first.outputs.BACKEND_HOST}}/actions/migrateSourceCredAccounts?force=true"
+          echo "::set-output name=DB_HOST::\
+          /${{env.DB_NAME}}?host=/cloudsql/${{steps.first.outputs.CLOUDSQL_CONNECTION_NAME}}"
+
+      - name: And That Result Again In Another Step
         id: third
         run: |
-          echo "::set-output name=GRAPHQL_URL::https://${{steps.second.outputs.HASURA_HOST}}/v1/graphql"
-          echo "::set-output name=BACKEND_TAG::${{steps.second.outputs.DOCKER_PATH}}/backend:pr-${{github.event.number}}"
-          echo "::set-output name=HASURA_TAG::${{steps.second.outputs.DOCKER_PATH}}/hasura:pr-${{github.event.number}}"
-          echo "::set-output name=FRONTEND_TAG::${{steps.second.outputs.DOCKER_PATH}}/frontend:pr-${{github.event.number}}"
-          echo "::set-output name=DB_PASSWORD::$(cat /dev/urandom | tr -cd [:alnum:] | head -c 16)"
+          echo "::set-output name=GRAPHQL_URL::\
+          https://${{steps.second.outputs.HASURA_HOST}}/v1/graphql"
+          echo "::set-output name=BACKEND_TAG::\
+          ${{steps.second.outputs.DOCKER_PATH}}/backend:pr-${{github.event.number}}"
+          echo "::set-output name=HASURA_TAG::\
+          ${{steps.second.outputs.DOCKER_PATH}}/hasura:pr-${{github.event.number}}"
+          echo "::set-output name=FRONTEND_TAG::\
+          ${{steps.second.outputs.DOCKER_PATH}}/frontend:pr-${{github.event.number}}"
 
   delete-db:
-    name: Delete Existing Database
+    name: Delete Hasura’s User & Database
     runs-on: ubuntu-latest
     needs: [env]
 
     steps:
-      - name: Set Up gcloud CLI
+      - name: Set Up GCloud CLI
```

Comment:
offical name is actually just "gcloud", all lowercase (according to its docs), same below